### ISSUE TITLE
Some population phenotypes are cellular phenotypes 

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4179]
Not sure if some of these are population phenotypes, since they represent the percentage of cells with a given cellular phenotype. Full list below, but some examples:

* Penetrance of cells that have the cellular phenotype bipolar growth
  * Decreased/ increased bipolar index
  * Decreased/increased frequency of apoptosis
  * 'decreased monopolar index'
  * 'increased number of cells with astral spindle microtubules'
* Not sure about some of these:
  * 
* 'abolished cell division timing change upon nitrogen source shift' probably should be under FYPO:0006820 ('abnormal cell growth rate')
* ``

<img width="496" alt="Screenshot 2022-06-29 at 13 13 19" src="https://user-images.githubusercontent.com/22526102/176423687-b94823a9-4774-4c90-8aa3-197666db4ea8.png">

<img width="474" alt="Screenshot 2022-06-29 at 13 15 34" src="https://user-images.githubusercontent.com/22526102/176423810-6fd17cc2-f66d-4306-9af5-f0f343b3408a.png">

**Comment by ValWood** [comment]
That's weird. The index terms  definitely were  always sadded as population terms

**Comment by ValWood** [comment]
I see you are suggesting they should not be.

I think this was decided  as a bit of a fudge because it is a population-level phenotype.

https://github.com/pombase/fypo/issues/49

it is the percentage of cells, but the  'index" is an increase/decrease in this percentage (not just the percentage)

**Comment by manulera** [comment]
Fair enough, I guess I will understand better the difference once I am familiar with logical definitions. In some cases where the same index can be caused by several cellular phenotypes (The septation index is a special could be altered by changes in the assembly/disassembly timing of the septum, or changes in the duration of the cell cycle).

For other cases where the population phenotype is `increased occurence of cell phenotype X`, I am not sure this is a population phenotype. `increased number of cells with astral spindle microtubules` is a clear case of this. I assume in this case what we are comparing is cells that are in mitosis, and comparing how often they have those astral microtubules, this phenotype is then more similar to `astral microtubules present in decreased numbers`, which is a cell phenotype.

**Comment by manulera** [comment]
In any case, maybe better for me to go through the uPheno materials before discussing this.

**Comment by manulera** [comment]
This could be closed, I think @ValWood
